org.eclipse.cdt.cross.arc.gnu/plugin.xml

```diff
@@ -745,6 +745,15 @@
             useByScannerDiscovery="false"
             valueType="boolean">
       </option>
+      <option
+            applicabilityCalculator="com.arc.cdt.toolchain.arc.ArcApplicabilityCalculator"
+            command="-L&quot;${VALUE}&quot;"
+            id="org.eclipse.cdt.cross.arc.gnu.base.option.linker.memoryx"
+            isAbstract="false"
+            name="memory.x file path"
```

Comment:
Sorry for lazy checking, but this is a **directory path**, not a file path. Also text would look better as **Directory with memory.x** - anything in the template of "path to memory.x", "directory with memory.x", etc.